editor/app/components/panel.jsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import {buildStyles, buildClassNames} from "./utils.js";
+
+export const PanelHeader = (props) => {
+  const {style, className, children} = props;
+  return (
+    <div
+      className={buildClassNames(["panel-header", className])}
+      style={buildStyles([styles.panelHeader, style])}
+    >
+      {children}
+    </div>
+  );
+};
+
+export const PanelBody = (props) => {
+  const {style, className, children} = props;
+  return (
+    <div
+      className={buildClassNames(["panel-body", className])}
+      style={buildStyles([styles.panelBody, style])}
```

Comment:
It's cool how you are allowing to use both APIs. There will be scenarios when we have to use CSS directly instead of JS objects and viceversa.

Reply:
Yeah Im leaving CSS objects support in place to see where this goes.  Its a super lightweight helper function for small tweaks. Will see how it works out mixing them.